mesonpy/__init__.py

```diff
@@ -133,40 +133,32 @@ def _init_colors() -> Dict[str, str]:
 assert all(re.match(_EXTENSION_SUFFIX_REGEX, x) for x in _EXTENSION_SUFFIXES)
 
 
-# Maps wheel installation paths to Meson installation path placeholders.
+# Map Meson installation path placeholders to wheel installation paths.
```

Comment:
It's maybe better if we refer to the wheel locations as schemes.

```suggestion
# Map Meson installation path placeholders to wheel installation schemes.
```

Reply:
Again, these are not schemes. See the `sysconfig` documentation, the wheel format documentation, and many other documents. In `sysconfig` nomenclature, "schemes" are the collections of the concrete paths where wheel components are installed. For example:
```
>>> sysconfig.get_scheme_names()
('nt', 'nt_user', 'nt_venv', 'osx_framework_user', 'posix_home', 'posix_prefix', 'posix_user', 'posix_venv', 'venv')
```
Each scheme defines "paths":
```
>>> sysconfig.get_path_names()
('stdlib', 'platstdlib', 'purelib', 'platlib', 'include', 'scripts', 'data')
```
"paths" is the thing we are dealing with here, not schemes.